src/packages/scalapb/update-hash.py

```
@@ -0,0 +1,62 @@
+    # Convert to SRI format
+    cmd_sri = ['nix-hash', '--to-sri', '--type', 'sha256', hash_legacy]
+    result_sri = subprocess.run(cmd_sri, capture_output=True, text=True)
+    return result_sri.stdout.strip()
+
+def update_nix_file(version, jar_hash):
+    import os
+    nix_file = os.path.join(os.path.dirname(__file__), 'default.nix')
+    
+    with open(nix_file, 'r') as f:
+        content = f.read()
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)